Find a functionally equivalent code:com.android.connectivitymanagertest.ConnectivityManagerTestBase.isConnectedToMobile:COMMENT
<com.android.connectivitymanagertest.ConnectivityManagerTestBase: boolean isConnectedToMobile()>
Method Modifier: public      protected   
Method Modifier: Yes         No          
Method Updated:  No          Yes         
Comment:// Return true if device is currently connected to mobile network

@@ -1,3 +1,3 @@
 {
-    return (mNetworkInfo.getType() == ConnectivityManager.TYPE_MOBILE);
+    return (mCm.getActiveNetworkInfo().getType() == ConnectivityManager.TYPE_MOBILE);
 }

